api-contracts/openapi/paths/webhook-worker/webhook-worker.yaml

```
@@ -0,0 +1,129 @@
+create:
+  post:
+    description: Creates a webhook
+    operationId: webhook:create
+    x-resources: ["tenant"]
+    parameters:
+      - description: The tenant id
+        in: path
+        name: tenant
+        required: true
+        schema:
+          type: string
+          format: uuid
+          minLength: 36
+          maxLength: 36
+    requestBody:
+      content:
+        application/json:
+          schema:
+            $ref: "../../components/schemas/_index.yaml#/WebhookWorkerCreateRequest"
+    responses:
+      "200":
+        content:
+          application/json:
+            schema:
+              $ref: "../../components/schemas/_index.yaml#/WebhookWorker"
+        description: Successfully created the webhook
+      "400":
+        content:
+          application/json:
+            schema:
+              $ref: "../../components/schemas/_index.yaml#/APIErrors"
+        description: A malformed or bad request
+      "401":
+        content:
+          application/json:
+            schema:
+              $ref: "../../components/schemas/_index.yaml#/APIErrors"
+        description: Unauthorized
+      "405":
+        content:
+          application/json:
+            schema:
+              $ref: "../../components/schemas/_index.yaml#/APIErrors"
+        description: Method not allowed
+    summary: Create a webhook
+delete:
+  post:
+    description: Deletes a webhook
+    operationId: webhook:delete
+    x-resources: ["tenant"]
```

```suggestion
    x-resources: ["tenant", "webhook"]
```

Note that when you add `webhook` to the path param here, you'll want to specify `webhook` as part of the resource hierarchy as well, which will automatically populate the tenant resource.